<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>511</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiempo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SRC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DST</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PROTO</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="buttonResumeCapture">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Continuar captura</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxInterfaces">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditRawData">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStopCapture">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener captura</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dispositivos de red disponibles:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Tipos de Filtrado:</string>
    </property>
    <widget class="QComboBox" name="comboBoxFilter">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>79</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>sinFiltro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UDP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>icmp</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="buttonSRCp">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SRC port</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textSRC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textSRCp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textDST">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDST">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DST</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSRC">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SRC</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Informaci√≥n del paquete:</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEditStructuredData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="labelId">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSrc">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDst">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTTL">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTOS">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>58</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSport">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDport">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFlags">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Id:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Source:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Destination:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>TTL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>TOS:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Src Port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Dst Port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Flags</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contenido RAW del paquete:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trafico capturado:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
